cli/src/swam/cli/Main.scala

```diff
@@ -340,25 +340,38 @@ object Main extends CommandIOApp(name = "swam-cli", header = "Swam from the comm
                           val functionName =
                           compiled.names.flatMap(_.subsections.collectFirstSome {
                             case FunctionNames(names) =>
+                              
+                              val mapList = names.values
+                              val check = mapList.exists(j => {j == func_name})
+                              if(!check){
+                                System.err.println("Function name does not exists.")
+                                sys.exit(1)
+                              }
                               names.get(typeIndex)
                             case _ =>
                               None
                         })
                         functionName match {
                           case Some(x) => {
                             if(x == func_name){
-                              println(s"This is function name : $func_name")
-                              println(s"This is param names : ${tpe.params}")
+                              val mapping = tpe.params.map(y => {
```

Comment:
The lambda body can be replaced by removing the `y =>` part

Reply:
Removed lambda body as part of the commit 2022710